portfolio/src/main/java/com/google/sps/servlets/DiscussionServlet.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
```

Comment:
Did the instructions say to include this in every file?  I think usually you can include this in a LISCENCE file in the root folder of the repo and it will cover everything, but im not sure what are policies are around here -- so default to whatever the instructions said

Reply:
I'm clarifying with the support team on this.

Reply:
Sounds good -- if they take a while to respond, I wouldn't let this hold up the cl...its definitely not a problem to include even if it isnt needed

Reply:
They said just having the LICENSE file in root should be okay - so I've went ahead and removed it from the Java file.